src/logic/map_objects/tribes/worker_program.cc

```diff
@@ -341,6 +344,8 @@
 			act->iparam2 = MapObjectDescr::get_attribute_id(item.second);
 		} else if (item.first == "type") {
 			act->sparam1 = item.second;
+		} else if (item.first == "name") {
+			act->sparam2 = item.second;
```

Comment:
actually I have no idea how to do that

Reply:
ok i found an implementation by using Std::find on the descriptions vector.  However I found out that only the descriptions with a register file are availabe at this stage. But this PR is explicitly about finding "hidden" mapObjects by their name. So e.g. a pinned note can be found by this PR but we cannot check its existence at this stage.   
So I hav no clue how to safeguard this other then using an assert in the run_findobject program to check for the existence in debug builds, as this would be very costly to check in each run of  a findobject program.    
  
Any Ideas?